src/config/env/prod.json

```diff
@@ -1,3 +1,6 @@
 {
-  "CHAIN_ID": "1"
-}
\ No newline at end of file
+  "CHAIN_ID": "1",
+  "ENVIRONMENT": "production",
+  "AUTH_URL": "/auth",
+  "SEGMENT_API_KEY": "sFdziRVDJo0taOnGzTZwafEL9nLIANZ3"
```

Comment:
can we pass this variables via the github actions ? I know they are already public but being exposed here feels weird.
Also why we need segment for this ? What if the user doesnt want to be tracked like we give the option in the sdk-commands ?

Reply:
Actually, `segment` is not required in the latest change. I'll remove it! thanks for the feedback